apps/native/app/src/screens/login/testing-login.tsx

```diff
@@ -113,6 +114,18 @@ export const TestingLoginScreen: NavigationFunctionComponent = ({
   }, [])
 
   const onLoginPress = async () => {
+    // Skip all login functionality if we are in mock environment
+    if (environment.id === 'mock') {
+      await setupNativeMocking()
```

Comment:
Let's dynamically import this function when needed to make sure none of the mocking code ends up in the main bundle. Can then use a static import of the polyfills, since they'll be dynamically imported from here.